lib/rules/web/admin_console/4.2/filters.xyaml

```diff
@@ -1,7 +1,7 @@
 set_filter_strings:
   elements:
   - selector: &filter_text_ele
-      css: input.text-filter
```

Comment:
Changes targeted for `Scenario: Check resource events`

Reply:
`input.text-filter`: match input, has attribute class, and the one of the class value **is exactly equal** to text-filter
`input[class*='text-filter']`: match input, has attribute class, and class value **contains** text-filter
And here's how the element is defined:
```
<input autocapitalize="none" class="pf-c-form-control co-text-filter" placeholder="Filter Events by name or message..." tabindex="0" type="text" value="">
```

Reply:
References: http://xahlee.info/js/css_selector_syntax.html